### PR TITLE
Remove inappropriate "please", minor rewording

### PR DESCRIPTION
**Pull request by BhaswatiRoy**
Fix: https://github.com/iterative/dvc.org/issues/3571

Rephrased all sentences in the documentation section that contained the "please" word into friendly sentences with similar meanings.

### CODE REVIEW

**Review by omesser — CHANGES_REQUESTED** [review]
Thanks @BhaswatiRoy !
lgtm 💯  - Just one typo

**Comment by BhaswatiRoy** [comment]
Thanks for pointing out @omesser 
I have fixed the typo

**Review by casperdcl — APPROVED** [review]
thanks!

File: content/docs/install/index.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ To check whether DVC is installed or which version you have, use `dvc version`.
 - You can [install a stable pre-release](/doc/install/pre-release) of DVC to
   stay ahead of official releases.
 
-- Shell completion is automatically enabled by certain installation methods. If
-  it does not work for you, please see
-  [these instructions](/doc/install/completion) to set it up.
+- Shell completion is automatically enabled by certain installation methods. If 
+  it does not work for you, you can refer to 
+  [these instructions](/doc/install/completion) to set it up
```
Comment (casperdcl) [inline content/docs/install/index.md:28]:
```suggestion
- Shell completion is automatically enabled by certain installation methods. If
  it does not work for you, see
  [these instructions](/doc/install/completion) to set it up.
```

File: content/docs/install/linux.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ In this case it installs the `boto3` library along with DVC.
 
 </details>
 
-> Please restart your terminal or re-source the shell configuration file
+> The shell configuration file needs to be restarted or re-sourced
```
Comment (casperdcl) [inline content/docs/install/linux.md:49]:
```suggestion
> Restart your terminal, or re-source the shell configuration file
```

File: content/docs/install/pre-release.md [hunk 3]
```diff
@@ -21,5 +21,5 @@ $ pip install "git+https://github.com/iterative/dvc#egg=dvc[s3]"
 > For information on our versioning convention, refer to
 > [Components of DVC version](/doc/command-reference/version#components-of-dvc-version).
 
-To install a development version for contributing to the project, please see
+To install a development version for contributing to the project, you can refer to the
 [Development environment](/doc/user-guide/contributing/core#development-environment).
```
Comment (casperdcl) [inline content/docs/install/pre-release.md:25]:
```suggestion
To install a development version for contributing to the project, see the
[Development environment](/doc/user-guide/contributing/core#development-environment)
guide.
```

File: content/docs/install/windows.md [hunk 4]
```diff
@@ -2,17 +2,16 @@
...
 </admon>
 
 <admon type="tip">
 
-Please review
-[Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows) for
-important tips to improve your experience using DVC on Windows.
-
+You can find important tips to improve your experience using DVC on Windows 
+by reviewing [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows).
```
Comment (casperdcl) [inline content/docs/install/windows.md:13]:
```suggestion
See [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows)
for important tips to improve your experience on Windows.
```

File: content/docs/start/experiment-management/experiments.md [hunk 5]
```diff
@@ -241,7 +241,7 @@ can specify text files to contain metrics using `dvc exp init --metrics`, and
 write them in the experimentation code.
 
 An alternative to manual metrics generation is to use [DVCLive](/doc/dvclive) to
-generate these files. Please refer to the documentation for details.
+generate these files. You can refer to the documentation for more details.
```
Comment (casperdcl) [inline content/docs/start/experiment-management/experiments.md:244]: redundant sentence

```suggestion
generate these files.
```

File: content/docs/start/experiment-management/experiments.md [hunk 6]
```diff
@@ -275,6 +275,6 @@ There are many other features of `dvc exp`, like cleaning up unused experiments,
 saving the current workspace status as an experiment, sharing them without
 committing into Git, or getting differences between two experiments.
 
-Please see the section on
+For additional information, you can refer to
```
Comment (casperdcl) [inline content/docs/start/experiment-management/experiments.md:278]:
```suggestion
For more information, see the section on
```

File: content/docs/studio/get-started.md [hunk 7]
```diff
@@ -14,7 +14,7 @@ your email address.
 
 When you sign up, you're on the **Free plan**. To switch to the **Basic plan**
 [create a team] first, then go to the _Team settings_ to [change the plan]. To
-sign up for the **Teams** or **Enterprise** plan, please [schedule a call] (see
+sign up for the **Teams** or **Enterprise** plan, you can [schedule a call] (see
```
Comment (casperdcl) [inline content/docs/studio/get-started.md:17]: intentional

```suggestion
sign up for the **Teams** or **Enterprise** plan, please [schedule a call] (see
```

File: content/docs/studio/troubleshooting.md [hunk 8]
```diff
@@ -38,7 +38,7 @@ Iterative Studio.
 
 ## Support
 
-If you need further help, please send us a message using `Help` on the
+If you need further help, feel free to send us a message using `Help` on the
```
Comment (casperdcl) [inline content/docs/studio/troubleshooting.md:41]:
```suggestion
If you need further help, you can send us a message using `Help` on the
```

File: content/docs/studio/troubleshooting.md [hunk 9]
```diff
@@ -163,7 +163,7 @@ There are two possible reasons for this:
    To create projects for subdirectories, [specify the project directory in
    project settings].
 
-   If this solution does not work for your use case, please create a support
+   If this solution does not work for your use case, don't hesitate to create a support
```
Comment (casperdcl) [inline content/docs/studio/troubleshooting.md:166]:
```suggestion
   If this solution does not work for your use case, please create a support
```

File: content/docs/studio/troubleshooting.md [hunk 10]
```diff
@@ -207,15 +207,15 @@ by you or someone else in your team.
 
 You can unhide commits and branches to display them. For details, refer to
 [Display preferences -> Hide commits]. However, if the missing commit/branch is
-not in the hidden commits list, then please [raise a support request](#support).
+not in the hidden commits list, then make sure to [raise a support request](#support).
```
Comment (casperdcl) [inline content/docs/studio/troubleshooting.md:210]:
```suggestion
not in the hidden commits list, please [raise a support request](#support).
```

File: content/docs/studio/troubleshooting.md [hunk 11]
```diff
@@ -207,15 +207,15 @@ by you or someone else in your team.
...
 
 [display preferences -> hide commits]:
   /doc/studio/user-guide/projects-and-experiments/explore-ml-experiments#hide-commits
 
 ## Error: Failed to push experiment to repository
 
 This is a non-specific error with a range of possible causes. To resolve it,
-please check:
+don't forget to check:
```
Comment (casperdcl) [inline content/docs/studio/troubleshooting.md:218]:
```suggestion
check that:
```

File: content/docs/studio/troubleshooting.md [hunk 12]
```diff
@@ -234,13 +234,13 @@ please check:
...
   are experiencing service disruptions. In case of an on-premises deployment,
-  please check with your administrator.
+  do check with your administrator.
 
 [gh-status]: https://www.githubstatus.com/
 [gl-status]: https://status.gitlab.com/
 [bb-status]: https://bitbucket.status.atlassian.com/
 
-If you get this error and none of the above applies, please
+If you get this error and none of the above applies, don't forget to
```
Comment (casperdcl) [inline content/docs/studio/troubleshooting.md:243]:
```suggestion
If you get this error and none of the above applies, please
```

File: content/docs/studio/troubleshooting.md [hunk 13]
```diff
@@ -285,7 +285,7 @@ Currently, you cannot un-assign stages from the Iterative Studio user interface.
 
 Check out the [Frequently Asked Questions](https://studio.iterative.ai/faq) to
 see if your questions have already been answered. If you still have problems,
-please [contact us](#support).
+don't hesitate to [contact us](#support).
```
Comment (casperdcl) [inline content/docs/studio/troubleshooting.md:288]:
```suggestion
please [contact us](#support).
```

File: content/docs/studio/user-guide/billing-and-payment.md [hunk 14]
```diff
@@ -34,7 +34,7 @@ Studio [pricing page](https://studio.iterative.ai/pricing).
 
 When you sign up, you're on the **Free plan**. To switch to the **Basic plan**
 [create a team] first, then go to the _Team settings_ to [change the plan]. To
-sign up for the **Teams** or **Enterprise** plan, please [schedule a call] (see
+sign up for the **Teams** or **Enterprise** plan, feel free to [schedule a call] (see
```
Comment (casperdcl) [inline content/docs/studio/user-guide/billing-and-payment.md:37]:
```suggestion
sign up for the **Teams** or **Enterprise** plan, please [schedule a call] (see
```

File: content/docs/studio/user-guide/billing-and-payment.md [hunk 15]
```diff
@@ -84,9 +84,9 @@ applied automatically for future payment.
 
 When you downgrade from the Basic plan to the Free plan, you will receive a
 prorated refund for the days remaining in the billing period. For inquiries
-about refunds, please [contact us].
+about refunds, don't hesitate to [contact us].
```
Comment (casperdcl) [inline content/docs/studio/user-guide/billing-and-payment.md:87]:
```suggestion
about refunds, please [contact us].
```

File: content/docs/studio/user-guide/billing-and-payment.md [hunk 16]
```diff
@@ -84,9 +84,9 @@ applied automatically for future payment.
...
 
-To upgrade to or downgrade from Teams or Enterprise plan, please [contact us].
+To upgrade to or downgrade from Teams or Enterprise plan, feel free to [contact us].
```
Comment (casperdcl) [inline content/docs/studio/user-guide/billing-and-payment.md:89]:
```suggestion
To upgrade to or downgrade from Teams or Enterprise plan, please [contact us].
```

File: content/docs/studio/user-guide/billing-and-payment.md [hunk 17]
```diff
@@ -141,4 +141,4 @@ you see your payment history and get a copy of your invoices.
 
 You can find answers to other frequently asked questions in the Iterative Studio
 [FAQ page](https://studio.iterative.ai/faq). If you have any other questions,
-please [contact us].
+don't hesitate to [contact us].
```
Comment (casperdcl) [inline content/docs/studio/user-guide/billing-and-payment.md:144]:
```suggestion
please [contact us].
```

File: content/docs/studio/user-guide/change-team-plan-and-size.md [hunk 18]
```diff
@@ -55,7 +55,7 @@ only 1 or 2 collaborators or pending invites. Then click on the `Downgrade`
 button.
 
 When you downgrade, you will receive a prorated refund for the days remaining in
-the billing period. For inquiries about refunds, please [contact us].
+the billing period. For inquiries about refunds, feel free to [contact us].
```
Comment (casperdcl) [inline content/docs/studio/user-guide/change-team-plan-and-size.md:58]:
```suggestion
the billing period. For inquiries about refunds, please [contact us].
```

File: content/docs/studio/user-guide/change-team-plan-and-size.md [hunk 19]
```diff
@@ -65,7 +65,7 @@ needs and suggest a suitable plan and pricing.
 
 ### If you are on the Teams or Enterprise plan
 
-For any changes to the plan, please contact your focal point at Iterative.
+For any changes to the plan, feel free to contact your focal point at Iterative.
```
Comment (casperdcl) [inline content/docs/studio/user-guide/change-team-plan-and-size.md:68]:
```suggestion
For any changes to the plan, please ask your contact person at Iterative.
```

File: content/docs/studio/user-guide/change-team-plan-and-size.md [hunk 20]
```diff
@@ -95,4 +95,4 @@ The total payable amount or credit balance will be displayed for you to
 
 ### If you are on the Teams or Enterprise plan
 
-For any changes to the plan, please contact your focal point at Iterative.
+For any changes to the plan, don't hesitate to contact your focal point at Iterative.
```
Comment (casperdcl) [inline content/docs/studio/user-guide/change-team-plan-and-size.md:98]:
```suggestion
For any changes to the plan, ask your contact person at Iterative.
```

File: content/docs/studio/user-guide/connect-custom-gitlab-server.md [hunk 21]
```diff
@@ -29,7 +29,7 @@ do the following:
 
 Once the connection is successful, all the repositories in this GitLab server
 will become available when you try to add a project in your team workspace. If
-you have any trouble setting up the connection to your GitLab server, please
+you have any trouble setting up the connection to your GitLab server, feel free to
```
Comment (casperdcl) [inline content/docs/studio/user-guide/connect-custom-gitlab-server.md:32]:
```suggestion
you have any trouble setting up the connection to your GitLab server, please
```

File: content/docs/use-cases/index.md [hunk 22]
```diff
@@ -2,7 +2,7 @@
 
 We provide short articles on common data science scenarios that DVC can help
 with or improve. Our use cases are not written to be run end-to-end like
-tutorials. For more general, hands-on experience with DVC, please see
+tutorials. For more general, hands-on experience with DVC, you can head over to
```
Comment (casperdcl) [inline content/docs/use-cases/index.md:5]:
```suggestion
tutorials. For more general, hands-on experience with DVC, see
```

File: content/docs/use-cases/index.md [hunk 23]
```diff
@@ -29,8 +29,8 @@ learning models, and you want to
 DVC is for you!
 
 > We keep reviewing our docs and will include interesting scenarios that surface
-> in the community. Please, [contact us](/support) if you need help or have
+> in the community. Feel free to, [contact us](/support) if you need help or have
```
Comment (casperdcl) [inline content/docs/use-cases/index.md:32]:
```suggestion
> in the community. [Contact us](/support) if you need help or have
```

File: content/docs/use-cases/index.md [hunk 24]
```diff
@@ -29,8 +29,8 @@ learning models, and you want to
...
 > suggestions!
 
-Please choose from the navigation sidebar to the left, or click the `Next`
+You can choose from the navigation sidebar to the left, or click the `Next`
```
Comment (casperdcl) [inline content/docs/use-cases/index.md:35]:
```suggestion
Choose from the navigation sidebar to the left, or click the `Next`
```

File: content/docs/use-cases/versioning-data-and-models/tutorial.md [hunk 25]
```diff
@@ -36,10 +36,10 @@ model file.
 > We have last tested this tutorial with Python 3.7.
 
 You'll need [Git](https://git-scm.com/) to run the commands in this tutorial.
-Also, if DVC is not installed, please follow these [instructions](/doc/install)
+Also, if DVC is not installed, you can follow these [instructions](/doc/install)
 to do so.
```
Comment (casperdcl) [inline content/docs/use-cases/versioning-data-and-models/tutorial.md:40]:
```suggestion
Also, [follow these instructions to install DVC](/doc/install) if it's not
already installed.
```

File: content/docs/use-cases/versioning-data-and-models/tutorial.md [hunk 26]
```diff
@@ -36,10 +36,10 @@ model file.
...
 
-> If you're using Windows, please review
+> If you're using Windows, you can review
 > [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows) for
 > important tips to improve your experience.
```
Comment (casperdcl) [inline content/docs/use-cases/versioning-data-and-models/tutorial.md:44]:
```suggestion
> See [Running DVC on Windows](/doc/user-guide/how-to/run-dvc-on-windows)
> for important tips to improve your experience on Windows.
```

File: content/docs/user-guide/data-management/importing-external-data.md [hunk 27]
```diff
@@ -95,7 +95,7 @@ $ dvc run -n download_file \
 ```
 
 ⚠️ DVC requires both SSH and SFTP access to work with remote SSH locations.
-Please check that you are able to connect both ways with tools like `ssh` and
+Don't forget to check that you are able to connect both ways with tools like `ssh` and
```
Comment (casperdcl) [inline content/docs/user-guide/data-management/importing-external-data.md:98]:
```suggestion
Check that you can connect both ways with tools like `ssh` and
```

File: content/docs/command-reference/remote/add.md [hunk 28]
```diff
@@ -272,7 +272,7 @@ $ dvc remote add -d myremote ssh://user@example.com/path
 > See `dvc remote modify` for a full list of SSH parameters.
 
 ⚠️ DVC requires both SSH and SFTP access to work with remote SSH locations.
-Please check that you are able to connect both ways with tools like `ssh` and
+Do check whether you are able to connect both ways with tools like `ssh` and
```
Comment (casperdcl) [inline content/docs/command-reference/remote/add.md:275]:
```suggestion
Check that you can connect both ways with tools like `ssh` and
```

File: content/docs/user-guide/data-management/importing-external-data.md [hunk 29]
```diff
@@ -163,7 +163,7 @@ $ dvc remote add myssh ssh://user@example.com
 $ dvc remote modify --local myssh password 'mypassword'
 ```
 
-> Please refer to `dvc remote modify` for more details like setting up access
+> You may refer to `dvc remote modify` for more details like setting up access
```
Comment (casperdcl) [inline content/docs/user-guide/data-management/importing-external-data.md:166]:
```suggestion
> Refer to `dvc remote modify` for more details like setting up access
```

File: content/docs/user-guide/data-management/large-dataset-optimization.md [hunk 30]
```diff
@@ -112,7 +112,7 @@ $ dvc config cache.type hardlink,symlink
 > Refer to `dvc config cache` for more details.
 
 Note that with this `cache.type`, your workspace files will be in read-only mode
-in order to protect the cache from corruption. Please refer to
+in order to protect the cache from corruption. You may refer to
```
Comment (casperdcl) [inline content/docs/user-guide/data-management/large-dataset-optimization.md:115]:
```suggestion
in order to protect the cache from corruption. Refer to
```

File: content/docs/user-guide/data-management/managing-external-data.md [hunk 31]
```diff
@@ -127,7 +127,7 @@ $ dvc run -d data.txt \
 ```
 
 ⚠️ DVC requires both SSH and SFTP access to work with remote SSH locations.
-Please check that you are able to connect both ways with tools like `ssh` and
+Don't forget to check that you are able to connect both ways with tools like `ssh` and
```
Comment (casperdcl) [inline content/docs/user-guide/data-management/managing-external-data.md:130]:
```suggestion
Check that you can connect both ways with tools like `ssh` and
```

File: content/docs/user-guide/data-management/managing-external-data.md [hunk 32]
```diff
@@ -184,7 +184,7 @@ The default <abbr>cache</abbr> is in `.dvc/cache`, so there is no need to set a
 custom cache location for local paths outside of your project.
 
 > Except for external data on different storage devices or partitions mounted on
```
Comment (casperdcl) [inline content/docs/user-guide/data-management/managing-external-data.md:186]:
```suggestion
> Exceptions to the above include: external data on different storage devices, and
partitions mounted on
```

File: content/docs/user-guide/data-management/managing-external-data.md [hunk 33]
```diff
@@ -184,7 +184,7 @@ The default <abbr>cache</abbr> is in `.dvc/cache`, so there is no need to set a
...
-> the same file system (e.g. `/mnt/raid/data`). In that case please setup an
+> the same file system (e.g. `/mnt/raid/data`). In that case you can setup an
 > external cache in that same drive to enable [file links] and avoid copying
```
Comment (casperdcl) [inline content/docs/user-guide/data-management/managing-external-data.md:188]:
```suggestion
> the same file system (e.g. `/mnt/raid/data`). In such cases, set up an
> external cache in the same drive to enable [file links] and avoid copying
```

File: content/docs/user-guide/how-to/run-dvc-on-windows.md [hunk 34]
```diff
@@ -21,7 +21,7 @@ details [here](/doc/install/ide-plugins#visual-studio-code)!
 ## POSIX-like command line experience
 
 The regular Command Prompt (`cmd`) in Windows will most likely not help you use
-DVC effectively, or follow the examples in our docs. Please avoid it. There's no
+DVC effectively, or follow the examples in our docs. Make sure to avoid it. There's no
 perfect solution, but here are some ideas:
```
Comment (casperdcl) [inline content/docs/user-guide/how-to/run-dvc-on-windows.md:25]:
```suggestion
DVC effectively, nor help you follow the examples in our docs. Here are some
alternatives:
```

File: content/docs/user-guide/how-to/run-dvc-on-windows.md [hunk 35]
```diff
@@ -97,7 +97,7 @@ directory, as explained in
 By default, DVC tries to use [Less](<https://en.wikipedia.org/wiki/Less_(Unix)>)
 as pager for the output of `dvc dag`. Windows doesn't have the `less` command
 available however. Fortunately, there is a easy way of installing it via
-[Chocolatey](https://chocolatey.org/) (please install the tool first):
+[Chocolatey](https://chocolatey.org/) (don't forget to install the tool first):
```
Comment (casperdcl) [inline content/docs/user-guide/how-to/run-dvc-on-windows.md:100]:
```suggestion
[Chocolatey](https://chocolatey.org). After installing Chocolatey, run:
```

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 36]
```diff
@@ -75,7 +75,7 @@ folder i.e. `gdrive://<base>/path/to/folder`. The base can be one of:
    browser address bar, for example
    `https://drive.google.com/drive/folders/0AIac4JZqHhKmUk9PDA`.
 
-   > \* Please note the
+   > \* Do note the
```
Comment (casperdcl) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:78]:
```suggestion
   > \* Note the
```

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 37]
```diff
@@ -245,11 +245,11 @@ and cannot be downloaded_ running a `dvc push` or `dvc pull` command, check that
 the `gdrive_acknowledge_abuse` option is set:
 
 ```cli
-$ dvc remote modify myremote gdrive_acknowledge_abuse true
+$ dvc remote modify myremote gdrive_acknowledge_abuse true
```
Comment (casperdcl) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:248]:
```suggestion
$ dvc remote modify myremote gdrive_acknowledge_abuse true
```

File: content/docs/user-guide/how-to/setup-google-drive-remote.md [hunk 38]
```diff
@@ -245,11 +245,11 @@ and cannot be downloaded_ running a `dvc push` or `dvc pull` command, check that
 the `gdrive_acknowledge_abuse` option is set:
 
 ```cli
-$ dvc remote modify myremote gdrive_acknowledge_abuse true
+$ dvc remote modify myremote gdrive_acknowledge_abuse true 
 ```
 
 </admon>
-> Please note our [Privacy Policy (Google APIs)](https://dvc.org/doc/user-guide/privacy).
+> Don't forget to note our [Privacy Policy (Google APIs)](https://dvc.org/doc/user-guide/privacy).
```
Comment (casperdcl) [inline content/docs/user-guide/how-to/setup-google-drive-remote.md:252]:
```suggestion
> Note our [Privacy Policy (Google APIs)](https://dvc.org/doc/user-guide/privacy).
```

File: content/docs/user-guide/troubleshooting.md [hunk 39]
```diff
@@ -27,7 +27,7 @@ be reached, causing this error.
 To solve this, it's often possible to increase the open file descriptors limit,
 with `ulimit` on UNIX-like system (for example `ulimit -n 1024`), or
 [increasing Handles limit](https://blogs.technet.microsoft.com/markrussinovich/2009/09/29/pushing-the-limits-of-windows-handles/)
-on Windows. Otherwise, please try using a lower `JOBS` value.
+on Windows. Otherwise, you can try using a lower `JOBS` value.
```
Comment (casperdcl) [inline content/docs/user-guide/troubleshooting.md:30]:
```suggestion
on Windows. Otherwise, try using a lower `JOBS` value.
```

**Review by casperdcl — CHANGES_REQUESTED** [review]

**Review by casperdcl — APPROVED** [review]

**Comment by jorgeorpinel** [comment]
Thanks again @BhaswatiRoy and @casperdcl 🙏🏼

**Pull request merged**